### ISSUE TITLE
[Enhancement] Is it possible to add xctest to html report or provide another command line?

### ISSUE THREAD

**Issue by luisxiaomai** [TitouanVanBelle/XCTestHTMLReport#41]
Hi,

Firstly  thanks for this great tool. 
But is it possible to add xctest to html report or provide another command line, because we usually run test both include unit test and ui test?

Thanks
Luis

**Comment by TitouanVanBelle** [comment]
Hi @luisxiaomai

Could you provide an example of the command line that you run?

**Comment by luisxiaomai** [comment]
@TitouanVanBelle , we use [xcpretty ](https://github.com/supermarin/xcpretty) to generate html report which include xctest and xcuitest logs. What I hope is that this tool can also include xctest result not only xcuitest result.

**Comment by TitouanVanBelle** [comment]
@luisxiaomai XCUITestHTMLReport makes use of the `-resultBundlePath` options that you can pass to `xcodebuild`. It allows `xcodebuild` to save the results of the tests at a specified path.

What I noticed is that if you simply test a UI Testing target with for example the following command
```
xcodebuild test -workspace Workspace.xcworkspace -scheme UITests -destination 'platform=iOS Simulator,name=iPhone 8,OS=11.2' -verbose -resultBundlePath TestResults
```
Then the `TestResults` folder will contain the results of the UI Tests

If you test a Unit Testing target with for example the following command
```
xcodebuild test -workspace Workspace.xcworkspace -scheme UnitTests -destination 'platform=iOS Simulator,name=iPhone 8,OS=11.2' -verbose -resultBundlePath TestResults
```
Then the `TestResults` folder will contain the results of the Unit Tests

Until now all good.

Now if you don't have any schemes for your test targets and only have a scheme for your application, you will probably run a command like the following that will test both your UI and Unit Testing targets

```
xcodebuild test -workspace Workspace.xcworkspace -scheme Application -destination 'platform=iOS Simulator,name=iPhone 8,OS=11.2' -verbose -resultBundlePath TestResults
```

The problem with that is that the `TestResults` folder will only contain the result of one of the two targets. If I'm not mistaken the tests ran last seems to erase the results of the tests ran first.

Here's an example. Here I'm running both UI and Unit Tests

```
$ xcodebuild test -workspace XCUITestHTMLReport.xcworkspace -scheme XCUITestHTMLReportSampleApp -destination 'platform=iOS Simulator,name=iPhone 8,OS=11.2' -verbose -resultBundlePath TestResults | xcpretty
2017-12-21 23:05:32.589 xcodebuild[81897:9840173] [MT] IDETestOperationsObserverDebug: (9C0EAD8E-E880-404F-9B02-0A32526CB549) Beginning test session XCUITestHTMLReportSampleAppUITests-9C0EAD8E-E880-404F-9B02-0A32526CB549 at 2017-12-21 23:05:32.589 with Xcode 9C40b on target <DVTiPhoneSimulator: 0x7fd4c06cc320> {
		SimDevice: iPhone 8 (F9328EF5-4D99-44E2-B91D-15A30D0216E4, iOS 11.2, Booted)
} (11.2 (15C107))
2017-12-21 23:05:32.593 xcodebuild[81897:9840227]  IDETestOperationsObserverDebug: Writing diagnostic log for test session to:
/var/folders/f_/43gkw_15049_531f5530w7dc0000gn/T/com.apple.dt.XCTest/IDETestRunSession-2BCAB27F-9805-4417-857B-49B50C7CF53B/XCUITestHTMLReportSampleAppUITests-0A0548C7-FB44-4CFA-B59F-9E2AAB9B23A3/Session-XCUITestHTMLReportSampleAppUITests-2017-12-21_230532-B4ONFd.log
2017-12-21 23:05:32.596 xcodebuild[81897:9840173] [MT] IDETestOperationsObserverDebug: (1FC2AE66-75AC-49F4-A0CF-E010B2755291) Beginning test session XCUITestHTMLReportSampleAppTests-1FC2AE66-75AC-49F4-A0CF-E010B2755291 at 2017-12-21 23:05:32.596 with Xcode 9C40b on target <DVTiPhoneSimulator: 0x7fd4c06cc320> {
		SimDevice: iPhone 8 (F9328EF5-4D99-44E2-B91D-15A30D0216E4, iOS 11.2, Booted)
} (11.2 (15C107))
2017-12-21 23:05:32.597 xcodebuild[81897:9840190]  IDETestOperationsObserverDebug: Writing diagnostic log for test session to:
/var/folders/f_/43gkw_15049_531f5530w7dc0000gn/T/com.apple.dt.XCTest/IDETestRunSession-2BCAB27F-9805-4417-857B-49B50C7CF53B/XCUITestHTMLReportSampleAppTests-C0641636-B3D4-4ED4-9097-880C8E732FFC/Session-XCUITestHTMLReportSampleAppTests-2017-12-21_230532-0m7a2G.log
▸ Building XCUITestHTMLReportSampleApp/XCUITestHTMLReportSampleApp [Debug]
▸ Check Dependencies
▸ Building XCUITestHTMLReportSampleApp/XCUITestHTMLReportSampleAppUITests [Debug]
▸ Check Dependencies
▸ Building XCUITestHTMLReportSampleApp/XCUITestHTMLReportSampleAppTests [Debug]
▸ Check Dependencies
▸ Compiling XCUITestHTMLReportSampleAppTests.swift
** TEST FAILED **

All tests
Test Suite XCUITestHTMLReportSampleAppUITests.xctest started
FirstSuite
    ✓ testOne (5.698 seconds)
    ✗ testTwo, XCTAssertTrue failed - Test failed
SecondSuite
    ✓ testOne (4.732 seconds)
    ✓ testTwo (4.771 seconds)


XCUITestHTMLReportSampleAppUITests.FirstSuite
  testTwo, XCTAssertTrue failed - Test failed
  FirstSuite.swift:56


	 Executed 4 tests, with 1 failure (0 unexpected) in 20.159 (20.164) seconds

All tests
Test Suite XCUITestHTMLReportSampleAppTests.xctest started
XCUITestHTMLReportSampleAppTests
    ✓ testExample (0.009 seconds)
    ✗ testPerformanceExample, XCTAssertTrue failed -


XCUITestHTMLReportSampleAppTests.XCUITestHTMLReportSampleAppTests
  testPerformanceExample, XCTAssertTrue failed -
  /Users/titouanvanbelle/workspace/personal/XCUITestHTMLReport/XCUITestHTMLReportSampleApp/XCUITestHTMLReportSampleAppTests/XCUITestHTMLReportSampleAppTests.swift:30
  ```
    {
        XCTAssert(false)
    }
  ```


	 Executed 2 tests, with 1 failure (0 unexpected) in 0.025 (0.027) seconds
```

**Comment by TitouanVanBelle** [comment]
@luisxiaomai I think I got what you want. I'm working on the fix. I let you know when it's released

**Comment by TitouanVanBelle** [comment]
@luisxiaomai 
It's ready but not released yet so you can try it out the following way

First remove you current `xchtmlreport` binary
```
rm /usr/local/bin/xchtmlreport
```

Then download a new one from `develop`
```
bash <(curl -s https://raw.githubusercontent.com/TitouanVanBelle/XCUITestHTMLReport/master/install.sh) 'develop'
```

Let me know if that works for you.

**Comment by TitouanVanBelle** [comment]
Released in 1.4.0

**Comment by luisxiaomai** [comment]
@TitouanVanBelle , that's what I want, Thanks for that.
